src/components/Card.tsx

```diff
@@ -0,0 +1,38 @@
+import { FunctionComponent } from "react";
+
+export interface ICardProps {
+  data: {
+    title: string;
+    description: string;
+    codePen: string;
+    github: string;
+  };
+}
+
+const Card: FunctionComponent<ICardProps> = ({ data }) => {
+  return (
+    <div className="card-wrap">
+      <div className="card">
+        <h3>{data.title}</h3>
+        <p className="card-description">
+          {data.description}
+          <br />
+          See code on{" "}
+          <a className="card-link" href={data.github}>

```

Comment:
Let's add `target="blank"` to links.

Reply:
- [ ] Done. [cf73870](https://github.com/AnastasiSo/Portfolio-Project/pull/22/commits/cf73870607ebc375f359fbf26865355ea07704fd)